diff --git a/dist/leaflet-src.js b/dist/leaflet-src.js
index c5b4c6d..64e5a2f 100644
--- a/dist/leaflet-src.js
+++ b/dist/leaflet-src.js
@@ -1356,6 +1356,10 @@ L.Map = L.Class.extend({

 		this._initContainer(id);
 		this._initLayout();
+
+		// hack for https://github.com/Leaflet/Leaflet/issues/1980
+		this._onResize = L.bind(this._onResize, this);
+
 		this.callInitHooks();
 		this._initEvents();
